Source/OxyPlot/Series/PieSeries.cs

```diff
@@ -258,51 +241,51 @@ public override void Render(IRenderContext rc)
                 return;
             }
 
-            this.total = this.slices.Sum(slice => slice.Value);
+            this.total = this.Slices.Sum(slice => slice.Value);
             if (Math.Abs(this.total) <= 0)
             {
                 return;
             }
 
             // todo: reduce available size due to the labels
-            double radius = Math.Min(this.PlotModel.PlotArea.Width, this.PlotModel.PlotArea.Height) / 2;
+            var radius = Math.Min(this.PlotModel.PlotArea.Width, this.PlotModel.PlotArea.Height) / 2;
```

Comment:
This whole block just more daunting without the types

Reply:
Yeah, with tooling it's certainly easier to interrogate, but it's the sort of thing where when scanning it's easy to be suspicious of every line after the first few because the types are unclear and so it distracts from the code itself (maybe that's just me). I'll live with it.